.github/workflows/eks-ci.yml

```diff
@@ -211,7 +211,7 @@ jobs:
 
       - name: Upload Test Results
         if: always()
-        uses: actions/upload-artifact@v4
+        uses: actions/upload-artifact@v3
```

Comment:
Please, also downgrade download-artifact to the same version

Reply:
@orestisfl, Opened a new [PR](https://github.com/elastic/cloudbeat/pull/1876) to revert the download-artifact action and also updated the dependabot.yaml to ignore 4.x versions.